### PR TITLE
[Linting] Fix linting errors

### LINKED ISSUE #4081: [Linting] Fix Lint Errors (will be required for merge)
#### Summary

We have 3 lint errors that need to be addressed before #4078 completed.

```
src/plugins/persistence/couch/CouchObjectProvider.js
50:66  error  '__OPENMCT_ROOT_RELATIVE__' is not defined  no-undef
```
```
src/plugins/remoteClock/RemoteClockSpec.js
140:93  error  Object properties must go on a new line  object-property-newline
```
```
src/plugins/remoteClock/plugin.js
25:64  error  Trailing spaces not allowed  no-trailing-spaces
```

#### Expected vs Current Behavior

No linting errors

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug?

#### Steps to Reproduce

1. Run `npm lint` at project root (for testing, confirm no errors, warnings ok for now)

### PR DESCRIPTION
**Pull request by jvigliotta**
### All Submissions:

Fixed a couple linting errors. closes #4081

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: src/plugins/persistence/couch/CouchObjectProvider.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ export default class CouchObjectProvider {
         let provider = this;
         let sharedWorker;
 
+        // eslint-disable-next-line no-undef
         const sharedWorkerURL = `${this.openmct.getAssetPath()}${__OPENMCT_ROOT_RELATIVE__}couchDBChangesFeed.js`;
```
Comment (shefalijoshi) [inline src/plugins/persistence/couch/CouchObjectProvider.js:51]: I don't think I understand this linting error. 

The definition for this in webpack is `__OPENMCT_ROOT_RELATIVE__: `'${devMode ? 'dist/' : ''`
So it should never be undefined, right?

**Review by shefalijoshi — COMMENTED** [review]
Small comment, but looks good.

File: src/plugins/persistence/couch/CouchObjectProvider.js (hunk 1 above)
Comment (davetsay) [reply src/plugins/persistence/couch/CouchObjectProvider.js:51]: I gotchu @jvigliotta 
https://eslint.org/docs/user-guide/configuring/language-options#specifying-globals

**Review by davetsay — APPROVED** [review]
LGTM. We should try to remove `eslint-disable-next-line` if possible but we can do this in another issue.

## Reviewer Checklist
| | |
| --- | :---: |
| Changes appear to address issue? | Y |
| Appropriate unit tests included? | N/A |
| Code style and in-line documentation are appropriate? | Y |
| Commit messages meet standards? | Y |
| Has associated issue been labelled `unverified`? | Y |

**Pull request merged**